docs/proteomics.md

```diff
@@ -0,0 +1,80 @@
+|------------|---------|------------|-------------------------|------------|-------------|-------|-------|---------------------------------------------------------------------------------------------------------------------|--------| 
+| CLUC       |         | CACO2      | LFQ.intensity.CACO2_1   | PROT_ANNOT | Colon       | LFQ-1 |       | Molecular profiling+High-throughput molecular profiling+Expression (protein)+LC-MS-MS+Protein level+ATTR1+MZ ratios | STD    | 
+| CLUC       |         | CACO2      | LFQ.intensity.CACO2_2   | PROT_ANNOT | Colon       | LFQ-2 |       | Molecular profiling+High-throughput molecular profiling+Expression (protein)+LC-MS-MS+Protein level+ATTR1+MZ ratios | STD    | 
+| CLUC       |         | COLO205    | LFQ.intensity.COLO205_1 | PROT_ANNOT | Colon       | LFQ-1 |       | Molecular profiling+High-throughput molecular profiling+Expression (protein)+LC-MS-MS+Protein level+ATTR1+MZ ratios | STD    | 
+| CLUC       |         | COLO205    | LFQ.intensity.COLO205_2 | PROT_ANNOT | Colon       | LFQ-2 |       | Molecular profiling+High-throughput molecular profiling+Expression (protein)+LC-MS-MS+Protein level+ATTR1+MZ ratios | STD    | 
+
+
+Data
+----
+
+The parameters file should be named `proteomics.params`.
+The [information provided for mRNA](expression.md) is applicable, with only some
+nomenclature changes.
+
+Example data file:
+
+| REF_ID | LFQ.intensity.CACO2_1 | LFQ.intensity.CACO2_2 | LFQ.intensity.COLO205_1 | LFQ.intensity.COLO205_2 | 
+|--------|-----------------------|-----------------------|-------------------------|-------------------------| 
+| 1860   | 9089400000            | 8792800000            | 8949100000              | 7252500000              | 
+| 2243   | 4997100000            | 5527800000            | 4280900000              | 4196200000              | 
+| 2739   | 2160700000            | 2090600000            | 30589000000             | 4188200000              | 
+| 2959   | 395180000             | 468840000             | 349410000               | 494790000               | 
+| 2260   | 101860000             | 84585000              | 93405000                | 101120000               | 
+| 4041   | 52779000              | 62863000              | 53180000                | 72288000                | 
+| 611    | 37153000              | 30627000              | 87144000                | 42039000                | 
+| 5060   | 0                     | 26186000              | 0                       | 0                       | 
+
+Format expectations:
+
+- First column has to be named `REF_ID` and contain row identifier. Row identifier has to match with `probesetID` from related platform.
+- All the rest of columns have to be intensities. There must not be other columns in the file.
+- The header names for intensities columns have to match with `sample_cd` values of the subject sample mapping file. 
```

Comment:
for **the** intensities columns have to match <del>with</del> *the* sample_cd` values of the subject sample mapping file.